.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: dart
+sudo: false
+dart:
+  - stable
+  - dev
+cache:
+  directories:
+    - $HOME/.pub-cache
+
+dist: trusty
```

Comment:
needed?

Reply:
not sure, cargo culted :)

Will try removing it